description.md

```diff
@@ -0,0 +1,7 @@
+#Zoom-bot
+The Zoom-bot is an interactive navigation and maintence interface that could be implemented on subways and trains all over the world to improve riders experience.
+The Zoom-bot is an interface to be placed on trains and subways. It should assist users by cleaning the train and giving them information including directions and air quality. The Zoom-bot will also keep track of train cleanliness and alert maintenance workers of problems. Subway employees will be using the interface to track trains’ cleanliness and location. 
+#Clinique Virtual Reality
+Clinique Virtual Reality is a way to experience personalized cosmetic advice from any location, at any time. It allows the user to scan their face, view the problem areas, and recommends products and solutions. 
```

Comment:
Nice work on the first two. My only question is what you actually created/delivered. Did you build full applications? Or just wireframes? Did you user test? Did you collaborate or work individually?